src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
```

Comment:
Use try / catch here & in all cases where you need to handle errors
```suggestion
  try {
    const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);

    return await response.json();
  } catch (e) {
    throw new Error('Something went wrong loading comments');
  }
```